guard/tests/test_command.rs

```diff
@@ -176,9 +181,14 @@ mod test_command_tests {
             .rules(Some(
                 "resources/validate/rules-dir/s3_bucket_server_side_encryption_enabled.guard",
             ))
+            .verbose(true)
             .run(&mut writer);
 
-        assert_eq!(0, status_code);
+        assert_eq!(StatusCode::SUCCESS, status_code);
+        assert_output_from_file_eq!(
+            "resources/test-command/output-dir/test_test_data_file.out",
```

Comment:
same as above, path should be 

`resources/test-command/output-dir/test_data_file.out`